.github/workflows/pr-test.yml

```diff
@@ -25,10 +25,10 @@ jobs:
         id: cache
         uses: actions/cache@v2
         with:
-          path: node_modules
+          path: ~/.npm
```

Comment:
path changed so have to update the package-lock to trigger a new cache to be installed at our new path